### ISSUE TITLE
Slider to use all available space

### ISSUE THREAD

**Issue by antonio1475** [piitaya/lovelace-mushroom#142]
**Is your feature request related to a problem? Please describe.**
I built a dashboard with conditional cards, so that when a light is on it shows the brightness slider.
![image](https://user-images.githubusercontent.com/63130724/154799736-51c61ac2-6726-4a35-8146-befbcc1e1ca7.png)

The slider needs more space, so it either expands the card(s), or I need to hide other cards.
![image](https://user-images.githubusercontent.com/63130724/154799753-992977b6-0789-44d1-8db2-ab1f15a8317a.png)

1.1.1 introduced "Horizontal layout", which is great. However, setting it to horizontal only uses 50% of the grid, even if everything else (name, state,...) is hidden. This makes the slider really hard to operate on mobile:
![image](https://user-images.githubusercontent.com/63130724/154799812-bb4ef359-5b80-40a9-b162-34057de20e16.png)

So, I played a bit more with css (card-mod) and finally:
![image](https://user-images.githubusercontent.com/63130724/154799825-a9ccc6ca-c074-449a-b2d2-26130d873c37.png)

**Describe the solution you'd like**
For better usability, especially in mobile, the slider to use all available space in the row, possibly by checking if name, state and whatever else information is disabled/empty.

I'm happy with my solution, so it's not really a request. Just sharing the feedback for your consideration for everyone, and sharing the idea with anyone who wants to do the same :) .

**Describe alternatives you've considered**
Adding this card-mod css code to the light card.
_Note: -50% worked great in desktop but overlapped with the icon on mobile (iPhone 13). -40% looks perfect on mobile, and also fantastic in desktop._

```
card_mod:
  style: >
    .container.horizontal > ::slotted(:not(:last-child)) {margin-right: -40%!important;}
```

Full card example:
```
type: custom:mushroom-light-card
entity: light.mesita
show_brightness_control: true
tap_action:
  action: toggle
hide_state: true
layout: horizontal
name: ''
card_mod:
  style: >
    .container.horizontal > ::slotted(:not(:last-child)) {margin-right: -40%!important;}
```

**Additional context**
Final video just for fun. Oh I love mushroom so much!!

https://user-images.githubusercontent.com/63130724/154799925-da39d771-330a-4a98-a59d-459ae1b673fc.mp4

**Comment by piitaya** [comment]
Thank for your feedback 🙂
The horizontal layout was developed to be used with a full width. 
I like your solution. We will considered this for a future version to improve usage on mobile. 
The editor feels a bit overloaded for now. We need to find a better organization (maybe tabs or accordions) before improving the cards and adding new options.

Thanks for sharing us a workaround for the current version ! 😃

**Comment by antonio1475** [comment]
Hi. Just popping in to mention that the code in the OP no longer works. I just realized but it's been like that for weeks.

Updated code (much simpler now) is:
```
card_mod:
  style: |
    mushroom-state-item {margin-right: -25%;}
```

It can also be done with max-right
And the number (% or px) needs to be adjusted based to your needs (devices size, names enabled/disabled, length of the names, etc). Just some quick trial & error does the trick.

![image](https://user-images.githubusercontent.com/63130724/174497952-4d45e9c7-4814-4b40-add2-69c883cd0e92.png)
